hkmc2/shared/src/main/scala/hkmc2/semantics/Elaborator.scala

```diff
@@ -1113,10 +1115,11 @@ extends Importer:
               
               val tsym = TermSymbol(k, owner, id) // TODO?
               val tdf = TermDefinition(k, sym, tsym, pss, tps, s, body, r, 
-                TermDefFlags.empty.copy(isMethod = isMethod), mfn, annotations, N)
+                TermDefFlags.empty.copy(isMethod = isMethod), mfn, N)
               tsym.defn = S(tdf)
               sym.defn = S(tdf)
-              
+              tsym.annotations = annotations
+              sym.annotations = annotations
```

Comment:
You should document that this is only a temporary measure while the situation with term symbols is messed up.